addon/services/key-manager.js

```diff
@@ -157,11 +175,21 @@ export default Ember.Service.extend({
   _combosWithKeys(combos) {
     const downKeys = get(this, 'downKeys');
     return combos.filter((combo) => {
-      const keys = get(combo, 'keys');
+      const keys = get(combo, 'keys').slice();
+
+      const verifyMeta = !metaKeys.any(k => keys.includes(k)) || get(this, 'metaKey');
+      const verifyControl = !controlKeys.any(k => keys.includes(k)) || get(this, 'ctrlKey');
+      keys.removeObjects(metaKeys.slice().concat(controlKeys));
```

Comment:
It seems like we're tracking modifier keys in a number of different ways:

0. in key-codes we have one listing for each name variation e.g., [meta](https://github.com/IcarusWorks/ember-key-manager/blob/master/addon/utils/key-codes.js#L74-L76)
0. we group all the modifier key codes in the modifier-key-codes util
0. at the top of this file we are mapping all possible names for the same keys
0. here we are merging all possible modifier key names into one array.

Is there a way to just group the keys in a single place in such a way that we can accomplish all the above?